docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Getting Started
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
+
+## Declaring a Task
```

Comment:
Defining? Creating?

Reply:
“Creating” doesn't look like an appropriate term for this kind of operation. “Defining” looks better, though.

```suggestion
## Defining a Task
```